### ISSUE TITLE
Remove aria-hidden on drag operation

### ISSUE THREAD

**Issue by aschempp** [contao/contao#8453]
```suggestion
			$dragHandle = '<button type="button" class="drag-handle">' . Image::getHtml('drag.svg', sprintf(is_array($labelCut) ? $labelCut[1] : $labelCut, $arrRow['id'])) . '</button>';
```

Unrelated to how it's implemented right now, I would suggest to remove `aria-hidden="true"` from buttons.

It does improve accessibility (on images,, icons or e. g. offscreen content) in certain cases, however it does remove accessibility from the curent scope and all the descendants and should be used with care.

### Warning: Do not use aria-hidden="true" on focusable elements.

Buttons are focusable.. 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-hidden

_Originally posted by @zoglo in https://github.com/contao/contao/pull/8400#discussion_r2155280816_

**Comment by leofeyer** [comment]
> _keyboard-navigation =/= screenreader_

But ARIA attributes apply to screen readers and not to keyboard navigation, don't they?

**Comment by zoglo** [comment]
I'm referring to `But the drag buttons can never be used via screen reader, so what is the point?`

The screenreader should read out the alt text of the image but due to `aria-hidden` it won't.
Hovering over the image won't be read with NVDA for example.

See the video below (Where I also remove the attribute):

https://github.com/user-attachments/assets/48865f34-cbd2-45da-9cfd-d8e05b90a5b0

**Comment by zoglo** [comment]
> But we want screen readers to ignore the button.

If that's the case, then we can simply leave it as is, because it's already fine by design.

**Comment by leofeyer** [comment]
As discussed in the call, we want to remove `aria-hidden="true"` from the drag handle.
